index.coffee

```diff
@@ -12,6 +12,28 @@ exports.RegistryV2 = legacy.RegistryV2
 LEGACY_DOCKER_VERSION = '1.10.0'
 DEFAULT_PROGRESS_BAR_STEP_COUNT = 50
 
+tryExtractDigestHash = (evt) ->
+	return null if !evt?
```

Comment:
This is redundant -- it's internal function and the way it's used guarantees it's won't ever receive null.